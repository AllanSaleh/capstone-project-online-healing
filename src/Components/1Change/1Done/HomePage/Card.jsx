import React from 'react';

const Card = ({ title, content, showIcon, icon, buttonText, showButton }) => (
  <div className="flex flex-col max-w-sm items-center shadow-lg py-8 px-16">
    {showIcon && <img src={icon} alt="" className="flex-initial pb-4" />}
    <h3 className="text-center text-title pb-1 capitalize">{title}</h3>
    <p className="text-opacity-50 text-black text-center text-paragraph pb-4">{content}</p>
    {showButton && (
      <button
        type="button"
        className="bg-blue-dark py-2 px-4 text-btn text-black hover:bg-opacity-50 transition duration-300 rounded-md"
      >
        {buttonText}
      </button>
    )}
  </div>
);

export default Card;
