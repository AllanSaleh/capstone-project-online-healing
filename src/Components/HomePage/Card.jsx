import React from 'react';
import { useHistory } from 'react-router-dom';

export default function Card({ title, content, showIcon, icon, buttonText, showButton }) {
  const history = useHistory();

  const GoToRoute = () =>
    history.push({
      pathname: '/Purchase',
      state: { ticket: title, price: content },
    });

  return (
    <div className="h-full w-full my-4 flex flex-col items-center justify-evenly shadow-lg rounded-lg md:w-1/4 p-4">
      {showIcon && <img src={icon} alt="" className="flex-initial mb-4" />}
      <h3 className="text-center text-xl  lg:text-3xl 2xl:text-title pb-1 capitalize">
        {title} Tickets
      </h3>
      <p className="text-opacity-50 text-black text-center text-sm md:text-md lg:text-paragraph py-4">
        {content}$
      </p>
      {showButton && (
        <button
          onClick={() => GoToRoute()}
          type="button"
          className="bg-blue-dark py-2 px-4 uppercase text-paragraph border text-black hover:bg-transparent hover:border-blue-dark hover:text-blue-dark  transition-all duration-200 rounded-md"
        >
          {buttonText}
        </button>
      )}
    </div>
  );
}
