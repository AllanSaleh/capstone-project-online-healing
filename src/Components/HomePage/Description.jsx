import React from 'react';
import { useTranslation } from 'react-i18next';
import { useHistory } from 'react-router-dom';

import Therapists from './Images/therapists.svg';

export default function Description({ user }) {
  const history = useHistory();
  const { t } = useTranslation();

  const GoToRoute = () => {
    if (user.login && user.complete) history.push('/Booking1');
    else if (!user.login) alert(t('HomePage.LoginAlert'));
    else alert(t('HomePage.CompleteAlert'));
  };

  return (
    <div className="text-left bg-yellow-primary h-section w-full px-sides m-">
      <div className="lg:text-5xl text-lg font-bold lg:w-7/12 lg:font-normal lg:pt-20 pt-20 uppercase">
        {t('HomePage.Therapist')}
      </div>
      <div className="pt-12 pb-16">
        <img src={Therapists} alt="therapists" />
      </div>
      <div className="lg:text-lg text-sm lg:w-8/12 lg:pb-20">{t('HomePage.TherapistInfo')}</div>
      <div>
        <button
          onClick={GoToRoute}
          type="button"
          className="uppercase mt-6 ml-4 py-2 px-6 lg:py-4 lg:px-6 lg:text-subtitle text-base rounded-md bg-blue-dark border hover:bg-white hover:text-blue-dark hover:border-blue-dark"
        >
          {t('HomePage.Book')}
        </button>
      </div>
    </div>
  );
}
