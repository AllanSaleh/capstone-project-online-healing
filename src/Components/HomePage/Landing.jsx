import React from 'react';
import { useHistory } from 'react-router-dom';

import HpIllustration from './Images/HP_illustration.svg';
import Background from './Images/Background.svg';

export default function Landing({ user }) {
  const history = useHistory();

  const GoToRoute = () => {
    if (user.login && user.complete) history.push('/Booking1');
    else if (!user.login) alert('Please LogIn First!');
    else alert('Please Complete Your Profile in Profile Settings First!');
  };

  return (
    <div className="h-firstsection mt-navbar">
      {/* background pattern */}
      <div className="absolute top-0 left-0 z-0">
        <img src={Background} alt="Background" />
      </div>

      {/* content */}
      <div className="flex lg:flex-row flex-col lg:justify-between lg:m-56 relative z-0">
        <div className="pt-20 pb-28 flex flex-col lg:transform lg:translate-y-0 transform translate-y-56">
          <div className="lg:text-4xl text-base pl-4">WE ARE HERE TO</div>
          <div className="lg:text-9xl  text-6xl pl-4">HELP</div>
          <div>
            <button
              onClick={() => GoToRoute()}
              type="button"
              className="mt-6 ml-4 py-2 px-6 lg:py-4 lg:px-6 lg:text-subtitle text-base rounded-md bg-blue-dark border hover:bg-white hover:text-blue-dark hover:border-blue-dark"
            >
              BOOK AN APPOINTMENT
            </button>
          </div>
        </div>
        <div className="transform -translate-y-72 w-64 lg:transform lg:-translate-y-0 lg:w-auto lg:ml-0 ml-16">
          <img src={HpIllustration} alt="HP_illustration" />
        </div>
      </div>
    </div>
  );
}
