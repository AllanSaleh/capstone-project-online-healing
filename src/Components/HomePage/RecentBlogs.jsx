import React from 'react';

import Tech from './Images/Tech.svg';
import Trauma from './Images/Trauma.svg';
import Prev from './Images/Prev.svg';
import Next from './Images/Next.svg';

export default function RecentBlogs() {
  const blogs = [
    { title: 'Mental Health & Technology', img: Tech },
    { title: 'Healing Trauma', img: Trauma },
    { title: 'Mental Health & Technology', img: Tech },
    { title: 'Healing Trauma', img: Trauma },
    { title: 'Mental Health & Technology', img: Tech },
    { title: 'Healing Trauma', img: Trauma },
  ];

  return (
    <div className="flex flex-col justify-evenly h-64 lg:h-section px-sides bg-blue-light">
      <div className="text-3xl lg:text-title">RECENT BLOGS</div>

      <div className="flex justify-between items-center">
        <button
          onClick={() => {
            const NewPos = document.getElementById('Menu').scrollLeft - 500;
            document.getElementById('Menu').scrollTo({
              left: NewPos,
              behavior: 'smooth',
            });
          }}
          type="button"
          className="mr-2 lg:mr-4 h-10 lg:h-12 w-10 lg:w-12 px-3 bg-white rounded-lg border-2 border-black hover:bg-blue-dark"
        >
          <img src={Prev} alt="Previous" />
        </button>

        <div className="relative h-40 lg:h-80 w-full overflow-hidden">
          <div
            id="Menu"
            className="h-44 lg:h-96 box-border whitespace-nowrap overflow-x-auto overflow-y-hidden"
          >
            {blogs.map((blog, i) => (
              <div
                onClick={(e) => alert(`Open blog number ${e.target.closest('.relative').id}`)}
                id={i}
                className="relative inline-block w-11/12 lg:w-2/6 h-40 lg:h-80 box-border mx-2 lg:mx-12"
              >
                <img
                  className="rounded-lg absolute h-full w-full cursor-pointer"
                  src={blog.img}
                  alt="Background"
                />

                <div className="w-24 lg:w-44 p-2 rounded-lg text-center whitespace-normal cursor-pointer absolute bottom-6 left-2 lg:bottom-8 lg:left-4 text-white text-xs lg:text-paragraph bg-black bg-opacity-50">
                  {blog.title}
                </div>
              </div>
            ))}
          </div>
        </div>

        <button
          onClick={() => {
            const NewPos = document.getElementById('Menu').scrollLeft + 500;
            document.getElementById('Menu').scrollTo({
              left: NewPos,
              behavior: 'smooth',
            });
          }}
          type="button"
          className="ml-2 lg:ml-4 h-10 lg:h-12 w-10 lg:w-12 px-3 bg-white rounded-lg border-2 border-black hover:bg-blue-dark"
        >
          <img src={Next} alt="Previous" />
        </button>
      </div>
    </div>
  );
}
