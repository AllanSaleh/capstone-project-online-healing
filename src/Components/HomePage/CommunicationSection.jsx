import React from 'react';

import Card from './Card';

import PhoneCell from './Images/phone-cell.svg';
import ChatIcon from './Images/chat-icon.svg';
import VideoCallIcon from './Images/video-call-icon.svg';

const CommunicationSection = () => (
  <div className=" md:h-section flex flex-col justify-evenly px-sides">
    <h1 className="sm:text-xl md:text-4xl xl:text-title pb-2 uppercase mb-2">
      we can communicate through
    </h1>
    <div className="flex flex-col md:flex-row items-center md:items-start md:justify-around">
      <Card
        showIcon
        icon={PhoneCell}
        title="voice call"
        content="Feeling ready to start a conversation? Give your therapist a voice call and talk your heart out!"
        showButton={false}
      />

      <Card
        showIcon
        icon={ChatIcon}
        title="chat"
        content="Need to talk to someone?Come have a chat with your therapist!"
        showButton={false}
      />

      <Card
        showIcon
        icon={VideoCallIcon}
        title="video call"
        content="For better experience therapists recommend video calls, but always remember that its a choice!"
        showButton={false}
      />
    </div>
  </div>
);

export default CommunicationSection;
