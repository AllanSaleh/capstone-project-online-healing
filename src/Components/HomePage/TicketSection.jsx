import React from 'react';
import { Link } from 'react-router-dom';

import Card from './Card';

const TicketSection = ({ user }) => (
  <div id="Tickets" className="flex flex-col justify-evenly px-sides lg:h-section py-8">
    <div className="pb-8">
      <h1 className="text-xl md:text-4xl xl:text-title uppercase">Purchase tickets</h1>
      <h3 className="text-xs md:text-lg lg:text-subtitle text-opacity-75 mb-4 md:my-2">
        purchase tickets that can be used to book appointments!
      </h3>
    </div>
    <div className="flex flex-col md:flex-row items-center justify-around w-full lg:h-1/3">
      <Card showIcon={false} title="5" content="10$" showButton buttonText="Purchase" user={user} />

      <Card
        showIcon={false}
        title="25"
        content="40$"
        showButton
        buttonText="Purchase"
        user={user}
      />

      <Card
        showIcon={false}
        title="50"
        content="70$"
        showButton
        buttonText="Purchase"
        user={user}
      />
    </div>
    <div
      className="flex flex-col justify-around items-center p-4 rounded-lg h-1/3 lg:mx-16"
      style={{ boxShadow: '0px 4px 12px 4px rgba(178, 171, 171, 0.25)' }}
    >
      <h1 className="text-2xl lg:text-3xl uppercase text-center py-4">are you a counselor?</h1>
      <p className="w-3/4 text-base md:text-md lg:text-subtitle text-justify md:text-center text-opacity-50">
        Interested in joining our mental health platform? You decide your schedule and how much you
        want to work, weâ€™ll take care of the client referrals and billing details!
      </p>

      <div className="text-center my-8">
        <Link to="/Requirements">
          <button
            type="button"
            className="bg-blue-dark py-2 px-4 uppercase text-paragraph border text-black hover:bg-transparent hover:border-blue-dark hover:text-blue-dark  transition-all duration-200 rounded-md"
          >
            learn more
          </button>
        </Link>
      </div>
    </div>
  </div>
);

export default TicketSection;
