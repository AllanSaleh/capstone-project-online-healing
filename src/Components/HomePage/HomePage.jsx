import React, { useState, useEffect } from 'react';
import { useLocation } from 'react-router-dom';

import Landing from './Landing';
import Description from './Description';
import CommunicationSection from './CommunicationSection';
import RecentBlogs from './RecentBlogs';
import TicketSection from './TicketSection';

const vh = Math.max(document.documentElement.clientHeight || 0, window.innerHeight || 0);

export default function HomePage({ loginStatus }) {
  const location = useLocation();
  const [blogs, setBlogs] = useState([]);

  useEffect(() => {
    if (location.state === true) {
      const y =
        document.getElementById('Tickets').getBoundingClientRect().top +
        window.pageYOffset -
        0.1 * vh;
      window.scrollTo({ top: y, behavior: 'smooth' });
    }
  }, []);

  return (
    <>
      <Landing user={loginStatus} />
      <Description user={loginStatus} />
      <CommunicationSection />
      <RecentBlogs blogs={blogs} setBlogs={(blogs) => setBlogs(blogs)} />
      <TicketSection user={loginStatus} />
    </>
  );
}
