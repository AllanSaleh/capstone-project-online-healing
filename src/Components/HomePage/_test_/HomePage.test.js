import * as React from 'react';
import HomePage from "../HomePage";
import * as renderer from "react-test-renderer";

it('Home Page renders correctly', () => {
  const tree = renderer
    .create(<HomePage/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});