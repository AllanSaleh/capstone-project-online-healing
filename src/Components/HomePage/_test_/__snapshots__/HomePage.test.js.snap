// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="h-firstsection  lg:-mt-0 mt-32"
  >
    <div
      className="absolute lg:top-2 top-16 left-0 z-0"
    >
      <img
        alt="Background"
        className="h-auto w-auto"
        src="Background.svg"
      />
    </div>
    <div
      className="flex lg:flex-row flex-col lg:justify-between lg:m-56 relative z-40"
    >
      <div
        className="pt-20 pb-28 flex flex-col lg:transform lg:translate-y-0 transform translate-y-56"
      >
        <div
          className="lg:text-4xl text-base pl-4"
        >
          WE ARE HERE TO
        </div>
        <div
          className="lg:text-9xl  text-6xl pl-4"
        >
          HELP
        </div>
        <div>
          <button
            className="mt-6 ml-4 py-2 px-6 lg:py-4 lg:px-6 lg:text-subtitle text-base rounded-md bg-blue-dark border hover:bg-white hover:text-blue-dark hover:border-blue-dark"
            type="button"
          >
            BOOK AN APPOINTMENT
          </button>
        </div>
      </div>
      <div
        className="transform -translate-y-72 w-64 lg:transform lg:-translate-y-0 lg:w-auto lg:ml-0 ml-16"
      >
        <img
          alt="HP_illustration"
          src="HP_illustration.svg"
        />
      </div>
    </div>
  </div>,
  <div
    className="text-left bg-yellow-primary h-section w-full px-sides m-"
  >
    <div
      className="lg:text-5xl text-lg font-bold lg:w-7/12 lg:font-normal lg:pt-20 pt-20 uppercase"
    >
      Professional, licensed, and vetted therapists that you can trust
    </div>
    <div
      className="pt-12 pb-16"
    >
      <img
        alt="therapists"
        src="therapists.svg"
      />
    </div>
    <div
      className="lg:text-lg text-sm lg:w-8/12 lg:pb-20"
    >
      Tap into the world's largest network of licensed, accredited, and experienced therapists who can help you with a range of issues including depression, anxiety, relationships, trauma, grief, and more. with our therapists, you get the same professionalism and quality you would expect from an in-office therapist, but with the ability to communicate whenever and however you want.
    </div>
    <div>
      <button
        className="mt-6 lg:py-4 lg:px-6  py-2 px-6 lg:text-subtitle text-base rounded-md bg-blue-dark border hover:bg-white hover:text-blue-dark hover:border-blue-dark"
        type="button"
      >
        LEARN MORE
      </button>
    </div>
  </div>,
  <div
    className=" md:h-section flex flex-col justify-evenly px-sides"
  >
    <h1
      className="sm:text-xl md:text-4xl xl:text-title pb-2 uppercase mb-2"
    >
      we can communicate through
    </h1>
    <div
      className="flex flex-col md:flex-row items-center md:items-start md:justify-around"
    >
      <div
        className="flex flex-col max-w-sm items-center shadow-lg rounded-lg h-full w-11/12 md:w-1/3 my-6 md:my-0 md:mx-2 p-2 lg:py-6 lg:px-12"
      >
        <img
          alt=""
          className="flex-initial mb-4"
          src="phone-cell.svg"
        />
        <h3
          className="text-center text-xl  lg:text-3xl 2xl:text-title pb-1 capitalize"
        >
          voice call
        </h3>
        <p
          className="text-opacity-50 text-black text-center text-sm md:text-md lg:text-paragraph py-4"
        >
          Feeling ready to start a conversation? Give your therapist a voice call and talk your heart out!
        </p>
      </div>
      <div
        className="flex flex-col max-w-sm items-center shadow-lg rounded-lg h-full w-11/12 md:w-1/3 my-6 md:my-0 md:mx-2 p-2 lg:py-6 lg:px-12"
      >
        <img
          alt=""
          className="flex-initial mb-4"
          src="chat-icon.svg"
        />
        <h3
          className="text-center text-xl  lg:text-3xl 2xl:text-title pb-1 capitalize"
        >
          chat
        </h3>
        <p
          className="text-opacity-50 text-black text-center text-sm md:text-md lg:text-paragraph py-4"
        >
          Need to talk to someone?Come have a chat with your therapist!
        </p>
      </div>
      <div
        className="flex flex-col max-w-sm items-center shadow-lg rounded-lg h-full w-11/12 md:w-1/3 my-6 md:my-0 md:mx-2 p-2 lg:py-6 lg:px-12"
      >
        <img
          alt=""
          className="flex-initial mb-4"
          src="video-call-icon.svg"
        />
        <h3
          className="text-center text-xl  lg:text-3xl 2xl:text-title pb-1 capitalize"
        >
          video call
        </h3>
        <p
          className="text-opacity-50 text-black text-center text-sm md:text-md lg:text-paragraph py-4"
        >
          For better experience therapists recommend video calls, but always remember that its a choice!
        </p>
      </div>
    </div>
  </div>,
  <div
    className="flex flex-col justify-evenly h-64 lg:h-section px-sides bg-blue-light"
  >
    <div
      className="text-3xl lg:text-title"
    >
      RECENT BLOGS
    </div>
    <div
      className="flex justify-between items-center"
    >
      <button
        className="mr-2 lg:mr-4 h-10 lg:h-12 w-10 lg:w-12 px-3 bg-white rounded-lg border-2 border-black hover:bg-blue-dark"
        onClick={[Function]}
        type="button"
      >
        <img
          alt="Previous"
          src="Prev.svg"
        />
      </button>
      <div
        className="relative h-40 lg:h-80 w-full overflow-hidden"
      >
        <div
          className="h-44 lg:h-96 box-border whitespace-nowrap overflow-x-auto overflow-y-hidden"
          id="Menu"
        >
          <div
            className="relative inline-block w-11/12 lg:w-2/6 h-40 lg:h-80 box-border mx-2 lg:mx-12"
            id={0}
            onClick={[Function]}
          >
            <img
              alt="Background"
              className="rounded-lg absolute h-full w-full cursor-pointer"
              src="Tech.svg"
            />
            <div
              className="w-24 lg:w-44 p-2 rounded-lg text-center whitespace-normal cursor-pointer absolute bottom-6 left-2 lg:bottom-8 lg:left-4 text-white text-xs lg:text-paragraph bg-black bg-opacity-50"
            >
              Mental Health & Technology
            </div>
          </div>
          <div
            className="relative inline-block w-11/12 lg:w-2/6 h-40 lg:h-80 box-border mx-2 lg:mx-12"
            id={1}
            onClick={[Function]}
          >
            <img
              alt="Background"
              className="rounded-lg absolute h-full w-full cursor-pointer"
              src="Trauma.svg"
            />
            <div
              className="w-24 lg:w-44 p-2 rounded-lg text-center whitespace-normal cursor-pointer absolute bottom-6 left-2 lg:bottom-8 lg:left-4 text-white text-xs lg:text-paragraph bg-black bg-opacity-50"
            >
              Healing Trauma
            </div>
          </div>
          <div
            className="relative inline-block w-11/12 lg:w-2/6 h-40 lg:h-80 box-border mx-2 lg:mx-12"
            id={2}
            onClick={[Function]}
          >
            <img
              alt="Background"
              className="rounded-lg absolute h-full w-full cursor-pointer"
              src="Tech.svg"
            />
            <div
              className="w-24 lg:w-44 p-2 rounded-lg text-center whitespace-normal cursor-pointer absolute bottom-6 left-2 lg:bottom-8 lg:left-4 text-white text-xs lg:text-paragraph bg-black bg-opacity-50"
            >
              Mental Health & Technology
            </div>
          </div>
          <div
            className="relative inline-block w-11/12 lg:w-2/6 h-40 lg:h-80 box-border mx-2 lg:mx-12"
            id={3}
            onClick={[Function]}
          >
            <img
              alt="Background"
              className="rounded-lg absolute h-full w-full cursor-pointer"
              src="Trauma.svg"
            />
            <div
              className="w-24 lg:w-44 p-2 rounded-lg text-center whitespace-normal cursor-pointer absolute bottom-6 left-2 lg:bottom-8 lg:left-4 text-white text-xs lg:text-paragraph bg-black bg-opacity-50"
            >
              Healing Trauma
            </div>
          </div>
          <div
            className="relative inline-block w-11/12 lg:w-2/6 h-40 lg:h-80 box-border mx-2 lg:mx-12"
            id={4}
            onClick={[Function]}
          >
            <img
              alt="Background"
              className="rounded-lg absolute h-full w-full cursor-pointer"
              src="Tech.svg"
            />
            <div
              className="w-24 lg:w-44 p-2 rounded-lg text-center whitespace-normal cursor-pointer absolute bottom-6 left-2 lg:bottom-8 lg:left-4 text-white text-xs lg:text-paragraph bg-black bg-opacity-50"
            >
              Mental Health & Technology
            </div>
          </div>
          <div
            className="relative inline-block w-11/12 lg:w-2/6 h-40 lg:h-80 box-border mx-2 lg:mx-12"
            id={5}
            onClick={[Function]}
          >
            <img
              alt="Background"
              className="rounded-lg absolute h-full w-full cursor-pointer"
              src="Trauma.svg"
            />
            <div
              className="w-24 lg:w-44 p-2 rounded-lg text-center whitespace-normal cursor-pointer absolute bottom-6 left-2 lg:bottom-8 lg:left-4 text-white text-xs lg:text-paragraph bg-black bg-opacity-50"
            >
              Healing Trauma
            </div>
          </div>
        </div>
      </div>
      <button
        className="ml-2 lg:ml-4 h-10 lg:h-12 w-10 lg:w-12 px-3 bg-white rounded-lg border-2 border-black hover:bg-blue-dark"
        onClick={[Function]}
        type="button"
      >
        <img
          alt="Previous"
          src="Next.svg"
        />
      </button>
    </div>
  </div>,
  <div
    className="flex flex-col justify-evenly px-sides"
  >
    <div
      className="pb-16 pt-8"
    >
      <h1
        className="text-xl md:text-4xl xl:text-title uppercase"
      >
        Purchase tickets
      </h1>
      <h3
        className="text-xs md:text-lg lg:text-subtitle text-opacity-75 mb-4 md:my-2"
      >
        purchase tickets that can be used to book appointments!
      </h3>
    </div>
    <div
      className="flex flex-col md:flex-row items-center md:items-start md:justify-around"
    >
      <div
        className="flex flex-col max-w-sm items-center shadow-lg rounded-lg h-full w-11/12 md:w-1/3 my-6 md:my-0 md:mx-2 p-2 lg:py-6 lg:px-12"
      >
        <h3
          className="text-center text-xl  lg:text-3xl 2xl:text-title pb-1 capitalize"
        >
          5 Tickets
        </h3>
        <p
          className="text-opacity-50 text-black text-center text-sm md:text-md lg:text-paragraph py-4"
        >
          10$
        </p>
        <button
          className="bg-blue-dark py-2 px-4 uppercase text-paragraph border text-black hover:bg-transparent hover:border-blue-dark hover:text-blue-dark  transition-all duration-200 rounded-md"
          type="button"
        >
          Purchase
        </button>
      </div>
      <div
        className="flex flex-col max-w-sm items-center shadow-lg rounded-lg h-full w-11/12 md:w-1/3 my-6 md:my-0 md:mx-2 p-2 lg:py-6 lg:px-12"
      >
        <h3
          className="text-center text-xl  lg:text-3xl 2xl:text-title pb-1 capitalize"
        >
          25 Tickets
        </h3>
        <p
          className="text-opacity-50 text-black text-center text-sm md:text-md lg:text-paragraph py-4"
        >
          40$
        </p>
        <button
          className="bg-blue-dark py-2 px-4 uppercase text-paragraph border text-black hover:bg-transparent hover:border-blue-dark hover:text-blue-dark  transition-all duration-200 rounded-md"
          type="button"
        >
          Purchase
        </button>
      </div>
      <div
        className="flex flex-col max-w-sm items-center shadow-lg rounded-lg h-full w-11/12 md:w-1/3 my-6 md:my-0 md:mx-2 p-2 lg:py-6 lg:px-12"
      >
        <h3
          className="text-center text-xl  lg:text-3xl 2xl:text-title pb-1 capitalize"
        >
          50 Tickets
        </h3>
        <p
          className="text-opacity-50 text-black text-center text-sm md:text-md lg:text-paragraph py-4"
        >
          70$
        </p>
        <button
          className="bg-blue-dark py-2 px-4 uppercase text-paragraph border text-black hover:bg-transparent hover:border-blue-dark hover:text-blue-dark  transition-all duration-200 rounded-md"
          type="button"
        >
          Purchase
        </button>
      </div>
    </div>
    <div
      className="flex flex-col justify-around my-16 px-4 lg:px-8 rounded-lg"
      style={
        Object {
          "boxShadow": "0px 4px 12px 4px rgba(178, 171, 171, 0.25)",
        }
      }
    >
      <h1
        className="text-2xl md:text-4xl lg:text-title uppercase mx-auto py-4"
      >
        are you a counselor?
      </h1>
      <p
        className="text-sm md:text-md lg:text-subtitle text-justify md:text-center text-opacity-50 my-4 "
      >
        Interested in joining our mental health platform? You decide your schedule and how much you want to work, we’ll take care of the client referrals and billing details!
      </p>
      <div
        className="text-center my-8"
      >
        <button
          className="bg-blue-dark py-2 px-4 uppercase text-paragraph border text-black hover:bg-transparent hover:border-blue-dark hover:text-blue-dark  transition-all duration-200 rounded-md"
          type="button"
        >
          learn more
        </button>
      </div>
    </div>
  </div>,
]
`;
